CppImport/src/visitors/ExpressionVisitor.h

```diff
@@ -160,11 +161,12 @@ class CPPIMPORT_API ExpressionVisitor : public clang::RecursiveASTVisitor <Expre
 		bool TraverseCXXStaticCastExpr(clang::CXXStaticCastExpr* castExpr);
 		bool TraverseCXXFunctionalCastExpr(clang::CXXFunctionalCastExpr* castExpr);
 
+		ClangAstVisitor* baseVisitor_{};
```

Comment:
Moving this to the public section seems like a bit of a hack to me. Why was this necessary?